extensions/wikia/CommunityPage/templates/contributorsModule.mustache

```diff
@@ -1,4 +1,4 @@
-<div class="module ContributorsModule">
+<div class="module contributorsModule">
```

Comment:
https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#class-and-id-naming-conventions
maybe contributors-module?